tests/installation/releasenotes.pm

```diff
@@ -17,10 +17,7 @@ use testapi;
 
 sub run() {
 
-    if (!check_screen('release-notes-button', 5)) {
-        record_soft_failure 'workaround missing release notes';
-        return;
-    }
+    assert_screen 'release-notes-button', fail_message => 'missing release notes';
```

Comment:
Missing release notes should have a bugref when it happens...we either need to include it in the test, or, we need to be sure we have a review process that doesn't leave softfails unreviewed for a weeks..